packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5750,8 +5825,8 @@ export default class Meeting extends StatelessWebexPlugin {
       }
     }
 
-    if (receiveShare !== undefined) {
-      this.mediaProperties.mediaDirection.receiveShare = receiveShare;
+    if (shareAudioEnabled !== undefined || shareVideoEnabled !== undefined) {
+      this.mediaProperties.mediaDirection.receiveShare = shareAudioEnabled || shareVideoEnabled;
```

Comment:
it's probably worth wrapping it with `!!` like this: `!!(shareAudioEnabled || shareVideoEnabled)`, because one of them might be undefined, for example if  shareAudioEnabled is false and shareVideoEnabled is undefined, the existing code would set receiveShare to undefined instead of false

Reply:
Agreed. Done.